prime-angular-frontend/src/app/shared/components/address-autocomplete/address-autocomplete.component.ts

```diff
@@ -32,40 +36,44 @@ export class AddressAutocompleteComponent implements OnInit {
   }
 
   public onAutocomplete(id: string) {
-    this.addressRetrieved = null;
     this.addressAutocompleteResource.retrieve(id)
-      .subscribe((response: AddressAutocompleteRetrieveResponse[]) => {
```

Comment:
@AnaisHebert did a bit of clean up in this method and dropped the use of a closure